contribute.md

```diff
@@ -220,53 +220,101 @@ commits](https://help.github.com/en/github/collaborating-with-issues-and-pull-re
   \#systemroles on Libera.chat, or ask on the PR/issue itself.
 
 ### Write a good commit message
-Here are a few rules to keep in mind while writing a commit message
 
-   1. Separate subject from body with a blank line
-   2. Limit the subject line to 50 characters
-   3. Capitalize the subject line
-   4. Do not end the subject line with a period
-   5. Use the imperative mood in the subject line
-   6. Wrap the body at 72 characters
-   7. Use the body to explain what and why vs. how
+This section covers the Conventional Commits format that system roles use to collect changelog and release roles automatically, and then give some general advices on writing commit messages.
+
+#### Conventional Commits format
+
+System roles follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format, there is a CI job that ensures that all commits follow the format.
+Conventional commits make it possible to generate CHANGELOG and update release version fully automatically.
+
+There is automation that collects commit messages into CHANGELOG and into RHEL release notes, you must ensure that you write the commits properly.
```

Comment:
```suggestion
There is automation that collects commit messages into CHANGELOG and into release notes, you must ensure that you write the commits properly.
```
I think we should not mention RHEL here.

Actually, for RHEL, we should really use the BZ Doc Text formats for Enhancement and Bug Fix - then we could use the commit/PR message directly.  We should do that next after the Conventional Commits work.

Reply:
Okay, I'll work on BZ doc text separately rather than doing all at once you're right.

Reply:
I applied your feedback and merged to link to this section from email sent to our org, let me know if you have any further concerns I can continue in a future PR.